adding namespace and docs

puja108 opened the pull request.

puja108 commented:
RFR @marians @webwurst

marians commented:
Is the idea to add this to the docs as a guide?

I would remove `THUX` from the title and headline, since it's no longer self explanatory as an abbreviation. Maybe add a sentence somewhere early to explain that we call this example application "Twitter Hot URIs Example - short THUX".

marians commented:
The docs content still explains the `resolver-scaler`, which isn't there any more I think.

marians commented:
Also `rebrow` is still mentioned in the text.

marians commented:
The component names in text and graphical overview differ in some parts.

marians commented on docs/index.md:
```diff
@@ -0,0 +1,88 @@
+
+Then, you can deploy the app by running:
+
+`kubectl apply --filename https://raw.githubusercontent.com/giantswarm/twitter-hot-urls-example/master/manifests-all.yaml`
```
I'd prefer a code block here instead of inline

puja108 commented:
Yes, I'd like to have this in docs as an example, will work on your fixes

puja108 commented:
Fixed according to @marians' comments. For `rebrow` I added that by default it is not deployed, I didn't remove it cause it's also in the overview.png

marians commented:
BTW, I am available to change the overview image :)

webwurst commented:
please remove `DEVNOTES.md` :)

webwurst commented:
see there for current version to create `manifests-all.yaml`:
https://github.com/giantswarm/kubernetes-prometheus/blob/e1830e749452d769cc804976033886e7ed89fbc3/development.md

at the end there is no `---` anymore. still not sure if i need to add an additional `\n`.

puja108 commented:
done

puja108 commented:
aded some last minute fixes ;)

puja108 commented:
btw, I saw that rebrow is still deployed
